src/lib.zig

```diff
@@ -210,6 +210,28 @@ pub fn serialize(comptime T: type, data: T, l: *ArrayList(u8)) !void {
     }
 }
 
+pub fn deserialize_slice(comptime T: type, serialized: []const u8, out: []T) !void {
+    if (try isFixedSizeObject(T)) {
+        var i: usize = 0;
+        const pitch = @sizeOf(T);
+        while (i < out.len) : (i += pitch) {
```

Comment:
that used to be `inline` in the calling code. That made no sense and yet it worked.